app/models/global_statistic.rb

```diff
@@ -0,0 +1,42 @@
+class GlobalStatistic < ApplicationRecord
+  validates_inclusion_of :singleton_guard, :in => [0]
+
+  def self.instance
+    first_or_create!(singleton_guard: 0)
+  end
+
+  MARINE_STATS = %w(
+    total_marine_protected_areas
+    total_marine_oecms_pas
+    total_ocean_pa_coverage_percentage
+    total_ocean_oecms_pas_coverage_percentage
+    total_ocean_area_protected
+    total_ocean_area_oecms_pas
+    national_waters_pa_coverage_percentage
+    national_waters_oecms_pas_coverage_percentage
+    national_waters_pa_coverage_area
+    national_waters_oecms_pas_coverage_area
+    high_seas_pa_coverage_percentage
+    high_seas_pa_coverage_area
+    national_waters_percentage
+    global_ocean_percentage
+  ).freeze
+  def self.marine_stats
+    instance.slice(*MARINE_STATS)
+  end
+
+  GREEN_LIST_STATS = %w(
+    green_list_perc
+    green_list_area
+    green_list_count
+  ).freeze
+  def self.green_list_stats
+    instance.slice(*GREEN_LIST_STATS)
+  end
+
```

Comment:
Do you think there should be an accessor style method here as well for land stats, with corresponding array of strings?

Reply:
Yeah there could be, although I don't see how it would be used at the moment?
Most of the stats (if not all) are accessed via the marine and greenlist ones and directly from a specific field in the table

Reply:
That's fine then, merging